Gemfile

```diff
@@ -1,55 +1,56 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 # Specify your gem's dependencies in geoblacklight.gemspec
 gemspec
 
-group :development, :test do
-  # See https://github.com/rails/rails/issues/32955
-  gem "thor", ">= 0.19.0"
```

Comment:
updating the engine_cart stuff included this on its own below, so it's not needed here